src/main/java/picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.java

```diff
@@ -57,12 +57,18 @@
             "appropriate. For other platforms and models, users should experiment to find what works best.")
     public int OPTICAL_DUPLICATE_PIXEL_DISTANCE = OpticalDuplicateFinder.DEFAULT_OPTICAL_DUPLICATE_DISTANCE;
 
+    @Option(doc = "This number is the maximum size of a set of duplicate reads for which we will attempt to determine " +
```

Comment:
@eitanbanks You'll have to change the "@Option" to "@Argument" when you rebase, since the Barclay PR is merged in.